src/doomtype.h

```diff
@@ -151,7 +153,8 @@ enum
 #define PRINT_HIGH				2				// critical messages
 #define PRINT_CHAT				3				// chat messages
 #define PRINT_TEAMCHAT			4				// chat messages from a teammate
-#define PRINT_LOG				5				// only to logfile
+#define PRINT_PRIVATECHAT		5				// private chat messages from another player
+#define PRINT_LOG				6				// only to logfile
```

Comment:
This delta to GZDoom code should be marked with a comment. I'd also not change `PRINT_LOG`, since that is from GZDoom. Or is it necessary that `PRINT_PRIVATECHAT` is `PRINT_TEAMCHAT+1`?

Reply:
I figured private chat messages would have greater importance than team chat messages, so it should been one level higher. Would changing `PRINT_PRIVATECHAT` so that it matched `PRINT_LOG` be acceptable?

Reply:
Ah, I see. In that case it's fine to raise `PRINT_LOG` to `6`. Just leave comments to mark that you added `PRINT_PRIVATECHAT` and that you raised `PRINT_LOG` to make room.